packages/extension/src/companion/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { browser } from 'webextension-polyfill-ts';
+import { themeModes } from '@dailydotdev/shared/src/contexts/SettingsContext';
+import { getCompanionWrapper } from './common';
+import App, { CompanionData } from './App';
+
+const renderApp = ({ ...props }: CompanionData) => {
+  const { settings } = props;
+  getCompanionWrapper().classList.add(themeModes[settings.theme]);
+
+  // Set target of the React app to shadow dom
+  ReactDOM.render(<App {...props} />, getCompanionWrapper());
+};
+
+browser.runtime.onMessage.addListener(async (request) => {
```

Comment:
Why do we need the `async` for?
Can't we do the destructuring in the function declaration instead of the line below?